common/src/main/java/com/skytouch/utility/Mapper.java

```diff
@@ -0,0 +1,17 @@
+package com.skytouch.utility;
+
+import org.dozer.DozerBeanMapper;
+
+public final class Mapper{
+
+    private static final DozerBeanMapper mapper = new DozerBeanMapper();
+
+    private Mapper() {
+    }
+
+    public static <T> T mapObject(Object source, Class<T> destinationClass) {
```

Comment:
Ntipick question:
What if you thread attempt to access to this method at the same time, would we have a race condition?. If so, what you can do to fix it?

Reply:
to solve this problem the method should be synchronized or i am not sure about this... what do you think about use a enum to define the utility?

Reply:
Please elaborate the enum idea. 
My advise is, check the framework's doc about thread safetiness. If it doesn't have any doc, implement a  race condition strategy to avoid sharing this utility at the same time with different threads.

Reply:
I read de documentation and the DozerBeanMapper.java class is thread safe.